src/op_mode/interfaces.py

```diff
@@ -368,16 +372,20 @@ def _format_show_summary(data):
 
 @catch_broken_pipe
 def _format_show_counters(data: list):
-    formatting = '%-12s %10s %10s %10s %10s'
-    print(formatting % ('Interface', 'Rx Packets', 'Rx Bytes', 'Tx Packets', 'Tx Bytes'))
+    formatting = '%-12s %10s %10s %10s %10s %10s %10s %10s %10s'
+    print(formatting % ('Interface', 'Rx Packets', 'Rx Bytes', 'Tx Packets', 'Tx Bytes', 'Rx Dropped', 'Tx Dropped', 'Rx Errors', 'Tx Errors'))
```

Comment:
This is python2 syntax - please use tabulate which is the preferred output method in the new op-mode scripts.